tests/_support/Helper/Acceptance/WooCommerce.php

```diff
@@ -18,6 +18,9 @@ class WooCommerce extends \Codeception\Module
 	 */
 	public function setupWooCommercePlugin($I)
 	{
+		// Set Store in Live mode i.e. not in "Coming Soon" mode.
+		$I->haveOptionInDatabase( 'woocommerce_coming_soon', 'no' );
```

Comment:
WooCommerce 9.6.0 sets its store in 'Coming Soon' mode by default, hence this change in tests to ensure the store is live for tests to run.